user/views.py

```diff
@@ -1,3 +1,175 @@
+                return JsonResponse({'MESSAGE' : 'EMAIL_FORM'}, status = 401)
+            elif not re.match(password_form, data['password']) :
+                return JsonResponse({'MESSAGE' : 'PASSWORD_FORM'}, status = 401)
+            elif User.objects.filter(email = data['email']).exists() :
+                return JsonResponse({'MESSAGE' : 'EMAIL_IN_USE'}, status = 401)
+            else :
+                User.objects.create(
+                    email         = data['email'],
+                    name          = data['name'],
+                    phone_number  = data['phone_number'],
+                    password      = bcrypt.hashpw(data['password'].encode('utf-8'),bcrypt.gensalt()).decode(),
+                    date_of_birth = data['date_of_birth']
+                )
+                return JsonResponse({'MESSAGE' : 'ACCOUNT_CREATED'}, status = 401)
+        except KeyError:
+            return JsonResponse({'MESSAGE' : 'KEY_ERROR'}, status = 401)
+
+class SignInView(View) :
+    def post(self, request) :
+        data = json.loads(request.body)
+        try :
+            if User.objects.filter(email = data['email']).exists() :
+                account = User.objects.get(email = data['email'])
+                if bcrypt.checkpw(data['password'].encode('utf-8'), account.password.encode('utf-8')) == True :
+                    return JsonResponse({'MESSAGE' : 'SUCCESS' , 'AUTHORIZATION' : jwt.encode({'id' : account.id}, SECRET_KEY, ALGORITHM).decode()}, status = 200)
+                else:
+                    return JsonResponse({'MESSAGE' : 'EMAIL_PASSWORD_INVALID'}, status = 400)
+            else:
+                return JsonResponse({'MESSAGE' : 'EMAIL_DOES_NOT_EXIST'}, status = 400)
+        except Exception as e:
+            return JsonResponse({'MESSAGE':f'ERROR {e}'}, status = 400)
+
+class GoogleAuth(View) :
+    def post(self, request) :
+        try :
+            data  = json.loads(request.body)
+            token           = data['AUTHORIZATION']
+            google_response = requests.get(f'https://oauth2.googleapis.com/tokeninfo?id_token={token}')
+            google_identity = google_response.json()
+            if google_identity['email_verified'] == 'true' :
+                name                   = google_identity['name']
+                email                  = google_identity['email']
+                image_url              = google_identity['picture']
+                google_login, created  = User.objects.get_or_create(email = google_identity['email'])
+                google_login.name      = google_identity['name']
+                google_login.image_url = google_identity['picture']
+                google_login.save()
+            return JsonResponse ({'MESSAGE' : 'GOOGLE_AUTH_SUCCESSFUL' , 'AUTHORIZATION' : jwt.encode({'id' : google_login.id}, SECRET_KEY, ALGORITHM).decode()}, status = 200)
+            if google_identity['email_verified'] == 'false' :
```

Comment:
```suggestion
            if not google_identity['email_verified']:
```
파이썬에서는 if 조건문 평가시, 반환 되는 값 자체가 boolean 일 경우나, Truthy, Falsy 한 값일 경우,  == False == None == 0등으로 평가하지 않고, 그 값을 그대로 사용합니다. 그 자체로도 True or False를 판단할 수 있기 때문입니다. False한 지 평가하실 때는 not 을 활용해보세요 :)